mani_skill/utils/scene_builder/replicacad/rearrange/scene_builder.py

```diff
@@ -10,21 +10,19 @@
 
 import numpy as np
 import sapien
+import sapien.physx as physx
 import torch
 import transforms3d
 
 from mani_skill import ASSET_DIR
-from mani_skill.utils.scene_builder.registration import register_scene_builder
-from mani_skill.utils.structs import Actor
+from mani_skill.utils.structs import Actor, Articulation
 from mani_skill.utils.building import actors
 
-from mani_skill.utils.scene_builder.replicacad import ReplicaCADSceneBuilder
+from ...replicacad import ReplicaCADSceneBuilder
```

Comment:
try to avoid relative imports like this unless it is a __init__.py file. Sometimes it is useful for users to copy an entire file and use it but that only works without relative imports

Reply:
removed